draft-mouris-cfrg-mastic.md

```diff
@@ -217,10 +219,56 @@ into Mastic in {{vidpf}}. The appendix includes supplementary material:
 {::boilerplate bcp14-tagged}
 
 This document uses the same conventions and definitions of {{Section 2 of
-!VDAF}}. The following functions are as defined therein:
-
-* `gen_rand(len: int) -> bytes`
-* TODO List all functions we use in alphabetical order.
+  !VDAF}}. The following basic functions are as defined therein:
+
+* `bool(val: Any) -> bool` converts a value `val` to a Boolean.
+* `byte(number: int) -> bytes` returns the encoding of the input `int` as a
+  byte.
+* `bytearray(source: Union[str, bytes, bytearray, Iterable[int]]) -> bytearray`
+  returns a mutable sequence of bytes.
+* `bytes(source: Union[str, bytes, bytearray, Iterable[int]]) -> bytes` returns
+  a immutable sequence of bytes.
```

Comment:
Similar to https://github.com/jimouris/draft-mouris-cfrg-mastic/pull/89/files?diff=unified&w=0#r1823303860, I don't think we need to define all Python functions we use. It's sufficient to say "algorithms in this document are specified in Python version 3.12".

Note that we make a few exceptions to this in the base draft, where folks have specifically asked to clarify something.

Likewise below.